### PR TITLE
APP-1845 fix crash in platinum when InspectDevice called before properly started

### PR DESCRIPTION
**Pull request by nickhingston**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "ThirdParty/Neptune"]
 	path = ThirdParty/Neptune
-	url = https://github.com/plutinosoft/Neptune.git
+url=https://github.com/naimaudio/Neptune.git
```
Comment (c0diq) [inline .gitmodules:3]: Platinum must point to Plutinosoft Neptune only

File: Source/Core/PltDatagramStream.cpp [hunk 2]
```diff
@@ -36,9 +36,9 @@
 |   includes
 +---------------------------------------------------------------------*/
 #include "PltDatagramStream.h"
-#include "NptLogging.h"
+//#include "NptLogging.h"
 
-NPT_SET_LOCAL_LOGGER("platinum.inputdatagramstream")
+//NPT_SET_LOCAL_LOGGER("platinum.inputdatagramstream")
```
Comment (c0diq) [inline Source/Core/PltDatagramStream.cpp:41]: These comments should go back in.

File: Source/Core/PltCtrlPoint.cpp [hunk 3]
```diff
@@ -1242,6 +1242,12 @@ PLT_CtrlPoint::InspectDevice(const NPT_HttpUrl& location,
                              NPT_TimeInterval   leasetime)
 {
     NPT_AutoLock lock(m_Lock);
+	
+	if (!m_Started) {
+		NPT_LOG_INFO_1("InspectDevice called before control point started.  location url: %s",
```
Comment (c0diq) [inline Source/Core/PltCtrlPoint.cpp:1247]: InspectDevice is called from ProcessSsdpMessage only. I am not sure I understand why it is important for the ctrl point to be started.

File: Source/Core/PltDeviceData.cpp [hunk 4]
```diff
@@ -188,7 +188,7 @@ PLT_DeviceData::GetIconUrl(const char* mimetype,
         // pick the biggest and better resolution we can
         if (icon.m_Width  >= m_Icons[i].m_Width  ||
             icon.m_Height >= m_Icons[i].m_Height ||
-            icon.m_Depth  >= m_Icons[i].m_Depth  ||
+            //icon.m_Depth  >= m_Icons[i].m_Depth  ||
```
Comment (c0diq) [inline Source/Core/PltDeviceData.cpp:191]: You could have 2 icons of same size but different resolutions and would not get the best one by removing this.

**Comment by nickhingston** [comment]
not convinced I meant to make this a PR on upstream - apologies!

**Pull request closed without merging**